eve/workers/openstack-terraform/terraform/common.tf

```diff
@@ -23,6 +23,21 @@ variable "nodes_count" {
   default = "2"
 }
 
+variable "offline" {
+  type    = bool
+  default = true
+}
+
+variable "proxy_ip" {
+  type    = string
+  default = "10.100.4.67"
```

Comment:
We cannot rely on ScalityCloud dns `proxy-cache` ? It's mandatory for security group rules right ?

Reply:
:+1: - the `dns_a_record_set` data source cannot resolve it, as you did for `subscription.rhsm.redhat.com`?

Reply:
Yes, sure, we could totally do that.